pkg/runtime/runtime.go

```diff
@@ -1578,3 +1617,12 @@ func (a *DaprRuntime) establishSecurity(sentryAddress string) error {
 func componentDependency(compCategory ComponentCategory, name string) string {
 	return fmt.Sprintf("%s:%s", compCategory, name)
 }
+
+// Function to convert string list into Map for easy lookup. (Set alternative)
+func convertToMap(list []string) map[string]struct{} {
+	m := make(map[string]struct{})
+	for _, item := range list {
+		m[item] = struct{}{}
+	}
+	return m
+}
```

Comment:
if using `sets.String` (from K8s api-machinery which we already depend on) this will just be `sets.NewString(list)`